framework.go

```diff
@@ -175,6 +175,32 @@ func (f *Framework) ApplyCluster(ctx context.Context, cluster *application.Clust
 		return nil, err
 	}
 
+	ns := cluster.GetNamespace()
+	cm_name := fmt.Sprintf("%s-app-operator-user-values", cluster.Name)
+
+	// Create a ConfigMap with the user values for app-operator
+	configMapData := make(map[string]string, 0)
+	values := `
+	service.operatorkit.resyncPeriod: 1m
+	`
```

Comment:
The whitespace here is important.
```suggestion
	values := `
service:
  operatorkit:
    resyncPeriod: 1m
`
```